ui/src/pages/GlobalHome.js

```diff
@@ -0,0 +1,293 @@
+        >
+          {this.props.intl.formatMessage(messages.howDoesItWork)}
+        </Button>
+      </div>
+    );
+  }
+
+  renderRespondersCard() {
+    return (
+      <Card className="globalhome-thirdpartycard">
+        <div>
+          <h3 className="mdc-typography--headline3">
+            <FormattedMessage {...messages.responders} />
+          </h3>
+          <p className="mdc-typography--body1">
+            <FormattedHTMLMessage {...messages.respondersHowToHelp} />
+          </p>
+        </div>
+        <CardActions>
+          <CardActionButtons>
+            <Button
+              className="pf-button-secondary"
+              raised
+              onClick={() => console.log('clicked!')}
+            >
+              {this.props.intl.formatMessage(messages.learnHow)}
+            </Button>
+          </CardActionButtons>
+        </CardActions>
+      </Card>
+    );
+  }
+
+  renderDevelopersCard() {
+    return (
+      <Card className="globalhome-thirdpartycard">
+        <div>
+          <h3 className="mdc-typography--headline3">
+            <FormattedMessage {...messages.developers} />
+          </h3>
+          <p className="mdc-typography--body1">
+            <FormattedHTMLMessage {...messages.developersHowToHelp} />
+          </p>
+        </div>
+        <CardActions>
+          <CardActionButtons>
+            <Button
+              className="pf-button-secondary"
+              raised
+              onClick={() => console.log('clicked!')}
+            >
+              {this.props.intl.formatMessage(messages.getStarted)}
+            </Button>
+          </CardActionButtons>
+        </CardActions>
+      </Card>
+    );
+  }
+
+  renderThirdPartyHelpCards() {
+    return (
+      <div className="globalhome-thirdpartywrapper">
+        <div className="endbars-headerline-wrapper" dir="ltr">
+          <span className="mdc-typography--overline endbars-headerline">
+            <FormattedMessage {...messages.youCanHelp} />
+          </span>
+        </div>
+        <Grid>
+          <Row>
+            <Cell desktopColumns={6}>{this.renderRespondersCard()}</Cell>
+            <Cell desktopColumns={6}>{this.renderDevelopersCard()}</Cell>
+          </Row>
+        </Grid>
+      </div>
+    );
+  }
+
+  renderRepoList() {
+    var cells = this.state.repos.map(repo => (
+      <Cell key={repo.repoId}>
+        <RepoCard repoInfo={repo} />
+      </Cell>
+    ));
+    return <Grid><Row>{cells}</Row></Grid>;
+  }
+
+  render() {
+    if (this.state.error) {
+      return <div>An error occurred</div>
+    }
+    if (!this.state.isLoaded) {
+      return <LoadingIndicator />;
+    }
+    return (
+      <div className="globalhome-wrapper">
+        {this.renderHeader()}
+        {this.renderRepoList()}
+        {this.renderThirdPartyHelpCards()}
+        <Footer />
+      </div>
+    );
+  }
+}
+
+/**
+ * RepoCard is a component for a card in the repo list.
+ */
+
+const repoCardMessages = defineMessages({
```

Comment:
Any reason to define this separately from `messages`?

Reply:
No, I guess not really.